templates/support/index.html

```diff
@@ -8,10 +8,9 @@
 <section class="p-strip is-deep is-bordered">
   <div class="row u-equal-height">
     <div class="col-7">
-      <h1>The Ubuntu Advantage</h1>
+      <h1>Ubuntu Advantage</h1>
       <h4>Open Source support for the enterprise</h4>
```

Comment:
Copy doc reads: Open Source for the enterprise

Reply:
Updated :+1: